crates/bevy_math/src/compass.rs

```diff
@@ -114,6 +137,18 @@ pub enum CompassOctant {
 }
 
 impl CompassOctant {
+    /// The list of all possible [`CompassOctant`] variants.
+    pub const VARIANTS: [CompassOctant; 8] = [
```

Comment:
Might be nice to test that this is in the same order as `from_index` (could we implement `from_index` like that?)